src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandlerUtil.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.api.*;
+
+@Slf4j
+public class RequestHandlerUtil {
+
+    /*
+     * get the outcome from
+     * @params clientUnder18, clientPassportedBenefit
+     * @return outcome (if outcome is not determined returns null (for more information)
+     *
+     */
+    public static Under18.Outcome getOutcomeFromAgeAndPassport(boolean clientUnder18, boolean clientPassportedBenefit){
+        Under18.Outcome outcome = null;
+        if (clientUnder18){
+            outcome = Under18.Outcome.ELIGIBLE;
```

Comment:
@patelila Shouldn't it be INELIGIBLE as per the output in ticket description?
<img width="704" alt="Screenshot 2023-07-28 at 12 56 15" src="https://github.com/ministryofjustice/cfe-crime/assets/195928/b02694b6-1ebe-43e0-9778-fa1bcce4d7cc">

Reply:
I think this is a mistake as in Miro it is eligible. See![Screenshot 2023-07-28 at 12 28 12](https://github.com/ministryofjustice/cfe-crime/assets/137802184/5cbe831c-a1fd-4295-8298-5df5d2666473)